azure/notebooks/Azure-MNMG-RF.ipynb

```diff
@@ -8,7 +8,7 @@
     "\n",
```

Comment:
<p>I think:</p><ol><li>Since you are adding features, also mention adding features somewhere in the workflow list. </li><li>Since you mention the workflow here, it makes sense to move the header Machine Learning Workflow further up in the notebook before the headers Data Cleanup and Add Interesting Features?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Added the "adding features" step</p><p> </p><p>I wanted it to be closer to the actually function calls instead of above, for an easier read. Not sure how much of this will make sense if introduced while talking about other things too.</p>